testing/error.go

```diff
@@ -0,0 +1,15 @@
+package testing
+
+func IsErrorEqual(a, b error) bool {
```

Comment:
```suggestion
func IsErrorEqual(wantErr, err error) bool {
```
Can we rename it like this? I feel it'll be more intuitive.

Reply:
Yes, done